components/nimbus/src/tests/test_persistence.rs

```diff
@@ -4,7 +4,7 @@
 
 use crate::enrollment::ExperimentEnrollment;
 use crate::persistence::*;
-use crate::{error::Result, Database, Experiment};
+use crate::{error::Result, persistence::Database, Experiment};
```

Comment:
Nit: is `Database` included in `persistence::*`?

Reply:
🤦